src/Microsoft.DotNet.CodeFormatting.Tests/Microsoft.DotNet.CodeFormatting.Tests.csproj

```diff
@@ -68,6 +68,7 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\System.Collections.Immutable.1.1.33-beta\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
     </Reference>
+    <Reference Include="System.ComponentModel.Composition" />
```

Comment:
It’s used because I need to be able to compile the test input code. I could try and go scrape the assembly off the computer running the tests somehow, but the easiest way (and most reliable I can figure out) is just to add a reference to the tests, and then use the loaded assembly as a MetadataReference. Without the assembly, the type won’t get loaded, then I can’t reliably get the full name of it.

From: Jared Parsons [mailto:notifications@github.com]
Sent: Monday, April 13, 2015 2:57 PM
To: dotnet/codeformatter
Cc: Chad Nedzlek
Subject: Re: [codeformatter] Readonly field formatter (#109)


In src/Microsoft.DotNet.CodeFormatting.Tests/Microsoft.DotNet.CodeFormatting.Tests.csproj<https://github.com/dotnet/codeformatter/pull/109#discussion_r28283468>:

> @@ -68,6 +68,7 @@

>        <SpecificVersion>False</SpecificVersion>

>        <HintPath>..\packages\System.Collections.Immutable.1.1.33-beta\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>

>      </Reference>

> +    <Reference Include="System.ComponentModel.Composition" />

Most of your attribute checking seems to be by name vs. by type. Given that is this reference needed anymore?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/codeformatter/pull/109/files#r28283468>.